File: nthelement.py
n = int(input())
a,b,c = input().split()
num = [a,b,c]
for i in range(0,n):
    t = i+2
    temp = 0
    while t>0:
        temp = temp+int(num[t])
        t = t-1
    num.append(temp)
print(num)

File: nthelement.c
#include <stdio.h>
int main() {
  int n;
  scanf("%d",&n);
  int num[n];
  for (int i = 0; i < 3; i++) {
    scanf("%d",&num[i] );
  }
  for (int i = 3; i < n; i++) {
    num[i]=0;
    for (int j = i-1; j >= i-3; j--) {
      num[i]+=num[j];
    }
  }
  printf("%d",num[n-1] );
}
